### PR TITLE
[MM-50527] Explicitly allow the Mattermost views access to write to the clipboard using Copy Link

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
When upgrading to Electron v23, it seems that a new permission was added to check if we were allowed to write to the clipboard using the Clipboard API. I think the permission has existed for a while (since it didn't come up in the Breaking Changes) but it must have been recently enforced (or a bug was fixed)

This PR makes sure that we allowed that permission when coming from a trusted source. I've also added the calls widget window as a trusted source since it asks to play media sometimes.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-50527

```release-note
NONE
```

### CODE REVIEW

**Review by hmhealey — APPROVED** [review]

**Review by cpoile — APPROVED** [review]
Thanks! We also need it for copying links from the call thread RHS in the calls expanded view popout, should that be included here as well? (it's `callsWidgetWindow.popout`)

**Comment by devinbinnie** [comment]
> Thanks! We also need it for copying links from the call thread RHS in the calls expanded view popout, should that be included here as well? (it's `callsWidgetWindow.popout`)

Yes it should! I'll add that.

File: src/main/app/initialize.ts [hunk 1]
```diff
@@ -402,9 +402,15 @@ function initializeAfterAppReady() {
...
         const callsWidgetWindow = WindowManager.callsWidgetWindow;
-        if (callsWidgetWindow && webContents.id === callsWidgetWindow.win.webContents.id) {
-            callback(true);
-            return;
+        if (callsWidgetWindow) {
+            if (webContents.id === callsWidgetWindow.win.webContents.id) {
+                callback(true);
+                return;
+            }
+            if (callsWidgetWindow.popOut && webContents.id === callsWidgetWindow.win.webContents.id) {
```
Comment (cpoile) [inline src/main/app/initialize.ts:410]: Tested it, it looks like it needs to be:
```suggestion
            if (callsWidgetWindow.popOut && webContents.id === callsWidgetWindow.popOut.webContents.id) {
```

**Review by cpoile — APPROVED** [review]
Thanks Devin!

**Review by cpoile — APPROVED** [review]

**Pull request merged**